third_party/httplib/httplib.hpp

```diff
@@ -317,16 +317,16 @@ namespace detail {
  */
 
 template <class T, class... Args>
-typename std::enable_if<!std::is_array<T>::value, std::unique_ptr<T>>::type
+typename std::enable_if<!std::is_array<T>::value, duckdb_wrapped::std::unique_ptr<T>>::type
```

Comment:
Maybe we can just use `duckdb::unique_ptr` if we are going to change all of this anyway